O365/message.py

```diff
@@ -756,20 +756,20 @@ def reply(self, to_all=True):
             return None
 
         message = response.json()
-        message_body_content_html  = message['body']['content']
+        message_body_content_html = message['body']['content']
 
         pattern = re.compile(r'<b>Sent:</b> (.*?)<br>')
 
-        def change_sent_time(match):
+        """Uses RegEx to extract the Sent time and attaches a timezone (self.protocol.timezone) to it"""
+        def change_sent_time_timezone(match):
             sent_time_str = match.group(1)
             sent_time = dt.datetime.strptime(sent_time_str, '%A, %B %d, %Y %I:%M:%S %p')
-            sent_time_new = sent_time.replace(tzinfo=dt.timezone.utc).astimezone(self.protocol.timezone)
-            return '<b>Sent:</b> {:%A, %B %d, %Y %I:%M:%S %p}<br>'.format(sent_time_new)
+            sent_time_as_timezone = (sent_time.replace(tzinfo=dt.timezone.utc)
+                            .astimezone(self.protocol.timezone))
```

Comment:
so close, tab spacing here is off though, the `.` should line up just after the `(`. I'm surprised your IDE didn't catch this